exception_reporter.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'telemetry_logger'
   spec.add_dependency 'dependency'
-  spec.add_dependency 'null_attr'
+  spec.add_dependency 'subst_attr'
```

Comment:
This won't work. These dependencies are prox-gem dependencies. They're not published and therefore the RubyGems infrastructure can't install them.